tex/gregoriotex.lua

```diff
@@ -214,7 +214,8 @@ local function init(arg, enable_height_computation)
   if enable_height_computation then
     new_line_heights = {}
 
-    local mcb_version = luatexbase.get_module_version('luatexbase-mcb')
+    local mcb_version = luatexbase.get_module_version and
+        luatexbase.get_module_version('luatexbase-mcb') or 9999
```

Comment:
There's probably some implicit Lua grouping that I'm not aware of, but I think explicit logical grouping should be used to make clear whether this is `a and (b or c)` or `(a and b) or c`.